### DISCUSSION TITLE
how do you get the commands pop-up in your demo?

### DISCUSSION THREAD

**Discussion by Russ-dh** [jglev/obsidian-carry-forward#38] [Q&A]
I've just installed Carry-forward, but have no idea how to get the command pop-up that I see in your demo. I assigned two hot keys, but these seem to do nothing. How do you get that pop-up where you can choose commands?

**Comment by churnish** [comment]
Are you referring to the [Command palette](https://help.obsidian.md/Plugins/Command+palette)?
